__tests__/Itinerary.tests.js

```diff
@@ -0,0 +1,17 @@
+const Itinerary = require('../src/Itinerary');
+
+describe('Itinerary', () => {
+    it('can be instantiated', () => {
+        expect(new Itinerary()).toBeInstanceOf(Object);
+    });
+
+    it('has a port property with an array of ports', () => {
+        const valencia = {};
+        const palma = {};
+        const itinerary = new Itinerary ([valencia, palma]);
+
+        expect(itinerary.ports).toEqual([valencia, palma]);
+    });
+});
+
```

Comment:
You never need more than one blank line at the end of a file, and sometimes none at all. This varies depending on your workplaces coding standards, but is never more than one

Whatever approach you use, it should be consistent throughout the project